enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/converter/MobileTokenConverter.java

```diff
@@ -216,6 +216,8 @@ private static Optional<Attribute> buildAttribute(final OperationTemplateParam t
                 buildImageAttribute(templateParam, params);
             case "PARTY_INFO" ->
                 buildPartyInfoAttribute(templateParam, params);
+            case "RESULT_TEXTS" ->
+                buildResultTextsAttribute(templateParam);
```

Comment:
I think we should allow same customization as for other types, so the `params` should be used as a parameter here.

Reply:
As @petrdvorak wrote, the format should be for example
```json
{
  "formData": {
    "title": "Confirm Payment",
    "message": "Hello,\nplease confirm following payment:",
    "resultTexts": {
      "success": "Payment of ${amount} ${currency} was confirmed",
      "reject": "Payment was rejected",
      "failure": "Payment approval failed"
    },
    "attributes": [
      {
        "type": "HEADING",
        "id": "operation.heading",
        "label": "Utility Payment"
      }
    ]
  }
}
```